### ISSUE TITLE
Request: provide the official ProtoBuf Ubuntu PPA and/or .deb packages of protoc

### ISSUE THREAD

**Issue by protobufel** [protocolbuffers/protobuf#75]
There is already a PPA for ProtoBuf, https://launchpad.net/ubuntu/+source/protobuf , but it has only 2.5.0-9  version. 

To have your official up-to-date PPA on Ubuntu featuring protoc compiler would be a big deal. Also, under the release section, you could publish debian and, maybe, rpm protoc packages, in addition to the already existing Windows' one.

**Comment by Zhomart** [comment]
Is there script in this repo to build deb package?

**Comment by Croydon** [comment]
Any chance that this is happening?

**Comment by Zhomart** [comment]
I've created deb package using fpm:

```
fpm \
    -s dir -t deb \
    -n "protobuf-dev" \
    -v "3.0.0-beta-1" \
    -p "poco-3.0.0-beta-1-armv7l.deb" \
    -C "./install" \
    --description "Google Protobuf Library" \
    --category libs \
    --provides libprotobuf-dev \
    --depends libc6 \
    --depends libstdc++6 \
    --exclude /usr/bin \
    --deb-no-default-config-files \
    usr/lib usr/include
```

Of course first you need to build protobuf.

**Comment by mfonville** [comment]
I packaged 3.0.2 with python3 support for ubuntu.
https://launchpad.net/~maarten-fonville/+archive/ubuntu/protobuf

**Comment by haon4** [comment]
It looks like https://launchpad.net/ubuntu/+source/protobuf will gradually pick up new protoc release, so this seems obsolete.

**Comment by AndreasBackx** [comment]
Could this be reopened? The latest version available is 3.6.1.3 which is from 08/12/2018, over a year old by now. I wouldn't exactly call this up-to-date.

**Comment by ktechmidas** [comment]
Once again Ubuntu is a million miles behind... 3.21.12 vs 31.2
